libs/sdk-ui-filters/src/AttributeFilter/AttributeFilterButton.tsx

```diff
@@ -655,8 +660,35 @@ export const AttributeFilterButtonCore: React.FC<IAttributeFilterButtonProps> =
                         )}
                     </MediaQuery>
                 }
+                ref={dropdownRef}
+                onOpenStateChanged={onDropdownOpenStateChanged}
                 body={
-                    isAllFiltered ? (
+                    props.renderBody ? (
```

Comment:
You could reduce some of the duplicities here and in the `AttributeDropdownBody` call by providing a default props.renderBody value. So if user sets it, call it, if not call the default, but with the exact same parameters.